var/spack/repos/builtin/packages/hwloc/package.py

```diff
@@ -43,6 +45,7 @@ class Hwloc(AutotoolsPackage):
     version('1.11.1', 'feb4e416a1b25963ed565d8b42252fdc')
     version('1.9',    '1f9f9155682fe8946a97c08896109508')
 
+    variant('nvml', default=True, description="Support NVML device discovery")
```

Comment:
Since `NVML` is shipped with the CUDA SDK nowadays, shouldn't this variant also pull the `depeds_on('cuda', ...)` dependency?